src/Common/src/System/Drawing/ColorTranslator.cs

```diff
@@ -0,0 +1,425 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Globalization;
+
+namespace System.Drawing
+{
+    /// <summary>
+    /// Translates colors to and from GDI+ <see cref='Color'/> objects.
+    /// </summary>
+    public static class ColorTranslator
```

Comment:
can you delete this file from System.Drawing.Common folder?

Reply:
It is still used by System.Drawing.Common for netcoreapp2.0